cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -99,6 +100,7 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import static io.strimzi.api.ResourceAnnotations.ANNO_STRIMZI_IO_SKIP_BROKER_SCALEDOWN_CHECK;
```

Comment:
Can you use the `Annotation.ANNO_STRIMZI_IO_SKIP_BROKER_SCALEDOWN_CHECK` here?

Reply:
Sure